wordpress-mobile/WordPress-Android
# Information Architecture: Sync various nav menu items to match Calypso.

Fixes #10658 

To test:
1. Open My Site,
2. Check that the menus now show "Site", "Pages", "Posts", "Design", and "Manage", like on the  screenshot below:

![Screen Shot 2019-11-07 at 18 54 09](https://user-images.githubusercontent.com/266376/68387698-7b600f00-0191-11ea-81f6-f0ca8ccf6811.png)


PR submission checklist:
